test/Libraries/CoreNodesTests/ListTests.cs

```diff
@@ -950,5 +950,206 @@ public static void Chop6()
 
             Assert.Throws<InvalidCastException>(() => { List.Chop(list, lengths); });
         }
+
+        [Test]
```

Comment:
I added additional test for GrouByFunction. Test for SortByFunction was created before me...